atd-vze/README.md

```diff
@@ -8,6 +8,8 @@ For resources and updates, see the [Vision Zero Crash Data System](https://githu
 
 ## Getting started
 
+We use [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm) to keep our `node` versions in sync with our environments. With `nvm` installed, run `nvm use` from this directory to activate the current `node` and `npm` version required for this project. If you don't want to use `nvm`, refer to the `.nvmrc` file for the `node` version you should install.
+
```

Comment:
stole this word for word from @johnclary in this [PR](https://github.com/cityofaustin/atd-moped/pull/688) 🤪

Reply:
I remember that I mentioned [fnm](https://github.com/Schniz/fnm) in that PR. Ended up switching over, and it works really well. Almost instant Node version change but I really don't think we are sweating fractions of seconds so... 🙃